utils/clusterfile_util.go

```diff
@@ -0,0 +1,59 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/alibaba/sealer/cert"
+	"github.com/alibaba/sealer/logger"
+
+	"io/ioutil"
+	"os"
+)
+
+func GetDefaultClusterFilePath(clusterFile string) (string, error) {
```

Comment:
https://github.com/alibaba/sealer/blob/main/common/common.go#L136  The common module already has those functions

Reply:
ok，i'll read first. thx